Doc/library/sys.rst

```diff
@@ -1533,7 +1573,7 @@ always available.
       :file:`Objects/lnotab_notes.txt` for a detailed explanation of how this
       works.
       Per-line events may be disabled for a frame by setting
-      :attr:`f_trace_lines` to :const:`False` on that frame.
+      :attr:`!f_trace_lines` to :const:`False` on that frame.
```

Comment:
We could possibly link to https://docs.python.org/3/reference/datamodel.html#frame-objects here, which provides some docs on this attribute

Reply:
Hm, yeah, that sounds like an improvement.

Reply:
But there is already a link to that section a few lines below:

> For more information on code and frame objects, refer to :ref:`types`.

How about adding some more references, like this:

```diff
diff --git a/Doc/library/sys.rst b/Doc/library/sys.rst
index a15bd01a2f..0b4abda060 100644
--- a/Doc/library/sys.rst
+++ b/Doc/library/sys.rst
@@ -1572,7 +1572,7 @@ always available.
       :file:`Objects/lnotab_notes.txt` for a detailed explanation of how this
       works.
       Per-line events may be disabled for a frame by setting
-      :attr:`!f_trace_lines` to :const:`False` on that frame.
+      :attr:`!f_trace_lines` to :const:`False` on that :ref:`frame <frame-objects>`.
 
    ``'return'``
       A function (or other code block) is about to return.  The local trace
@@ -1591,7 +1591,7 @@ always available.
       ``None``; the return value specifies the new local trace function.
       Per-opcode events are not emitted by default: they must be explicitly
       requested by setting :attr:`!f_trace_opcodes` to :const:`True` on the
-      frame.
+      :ref:`frame <frame-objects>`.
 
    Note that as an exception is propagated down the chain of callers, an
    ``'exception'`` event is generated at each level.
```

Reply:
That LGTM!